### ISSUE TITLE
[Bug]:  fatal error: 'share/ob_master_key_getter.h' file not found

### ISSUE THREAD

**Issue by watchpoints** [oceanbase/oceanbase#1421]
**Describe the bug**

**Environment**

- OS Version and CPU Arch(`uname -a`):
`
 uname -a
Linux localhost.localdomain 4.18.0-448.el8.x86_64 #1 SMP Wed Jan 18 15:02:46 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
`
  
- OB Version(`LD_LIBRARY_PATH=../lib:$LD_LIBRARY_PATH ./observer -V`):
~~~
  usr/local/bin/observer -V
observer (OceanBase_CE 4.2.0.0)
REVISION: 1-88bcf89dbd6db244b26f1ce08eb987b7ca5a3d54
BUILD_BRANCH: master
BUILD_TIME: Jun  3 2023 21:38:14
BUILD_FLAGS: Debug
BUILD_INFO:
~~~

**Fast Reproduce Steps(Required)**

 cd ~/build_debug/unittest/share
 make 

**Expected behavior**

make succed

**Actual Behavior**

`mnt/oceanbase/oceanbase/unittest/share/backup/test_backup_struct.cpp:20:10: fatal error: 'share/ob_master_key_getter.h' file not found
#include "share/ob_master_key_getter.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
`
**Additional context**

**Comment by watchpoints** [comment]
where the file ob_master_key_getter.h

**Comment by longdafeng** [comment]
@hnwyllmm  please take a look at this issue.

**Comment by hnwyllmm** [comment]
fixed by commit a27900463108bda38b1991d637f4d9e2b0cb460d
